Cargo.toml

```diff
@@ -18,12 +18,13 @@ ipnetwork = "0.15.0"
 tui = "0.5"
 termion = "1.5"
 structopt = "0.3"
-dns-lookup = "1.0.1"
 signal-hook = "0.1.10"
 failure = "0.1.6"
 chrono = "0.4"
 regex = "1.3.1"
 lazy_static = "1.4.0"
+tokio = "0.2.0-alpha.6"
+trust-dns-resolver = "0.18.0-alpha.1"
```

Comment:
Can we use the stable version rather than the alpha one of these, or do we need features from the alpha?

Reply:
I've tried to downgrade both libs but it seems that those alphas are the only available versions compatible with async/await and `std::future::Future`. Both of them build properly in the stable toolchain though. Do you think we can keep them?

Reply:
That's the price of being on the edge, I guess. :) Let's keep 'em